### ISSUE TITLE
Problem with geofence and multiserver

### ISSUE THREAD

**Issue by turin400** [domoticz/domoticz-android#632]
Hi I am using two servers for example server_1 and server_2, I have two switches associated with two different geofences, switch_1 and switch_2. The problem is that when I am in the geofence of server_2, switch_1 is activated because switch_1 and switch_2 have the same idx but different server.

I do not know if I explain myself very well. With anything he tells me. 
All the best.

version 0.2.302(8479), android 10, galaxy s9.

**Comment by galadril** [comment]
Thats true, actually all automation options (speech/bluetooth/beacon/geofences/wifi/etc) they all assume your connected to the server that they need to toggle..

**Comment by turin400** [comment]
I had set up two geofences and thought they were independent for each server.
What was happening to me was that when I went to the second geofence, the air conditioning in my house was turning on and I was going crazy.
Is it possible that in some version this can be improved?
Especially the geofences, bluetooth, beacon, wifi, which are automatic and you do not have the phone in hand when they are triggered.

All the best.
